frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -157,14 +234,14 @@ const keywordCompare = (filterString, item) => {
   );
 };
 
-const setURLParams = (params) => {
-  const url = new URL(window.location);
-  const searchParams = `?${params.toString()}${url.hash}`;
+const setURLParams = (params): void => {
+  const url = new URL(window.location.toString());
+  const searchParams: string = `?${params.toString()}${url.hash}`;
```

Comment:
Just FYI you don't need a type for assignments. Only when the right side is ambiguous like `{...}`